src/dynamic-page-layout.scss

```diff
@@ -0,0 +1,361 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-dynamic-page-layout;
+
+.#{$block} {
+
+  $fd-dynamic-page-layout-background-color: var(--sapObjectHeader_Background);
+  $fd-dynamic-page-layout-content-background-color: var(--sapBackgroundColor);
+  $fd-dynamic-page-layout-padding-top-bottom: 0.5rem;
+  $fd-dynamic-page-layout-header-padding-top-bottom: 1rem;
+
+  @include fd-reset();
+
+  @mixin responsive-page($padding-y: 0,$padding-x: 1rem) {
+    padding: $padding-y $padding-x;
+  }
+
+  @mixin box-shadow() {
+    box-shadow: var(--sapContent_HeaderShadow);
+    // box-shadow: 0 0.25rem 0.25rem -0.25rem rgba(0,0,0,0.15), inset 0 -0.0625rem 0 0 var(--sapObjectHeader_BorderColor);
+  }
```

Comment:
Why is this a mixin? Could be just a variable

Reply:
removed mixin usage, moved to a common variable and used normally.